go-console/.vscode/tasks.json

```diff
@@ -0,0 +1,919 @@
+        "color": "terminal.ansiYellow"
+      }
+    },
+    {
+      "label": "pull-container-torizon-debug-arm",
+      "detail": "hide",
+      "command": "sshpass",
+      "type": "process",
+      "args": [
+        "-p",
+        "${config:torizon_psswd}",
+        "ssh",
+        "-o",
+        "UserKnownHostsFile=/dev/null",
+        "-o",
+        "StrictHostKeyChecking=no",
+        "torizon@${config:torizon_ip}",
+        "LOCAL_REGISTRY=${config:host_ip} TAG=arm docker-compose pull __container__-debug"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "flame",
+        "color": "terminal.ansiYellow"
+      }
+    },
+    {
+      "label": "run-container-torizon-debug-arm",
+      "detail": "hide",
+      "command": "sshpass",
+      "type": "process",
+      "args": [
+        "-p",
+        "${config:torizon_psswd}",
+        "ssh",
+        "-o",
+        "UserKnownHostsFile=/dev/null",
+        "-o",
+        "StrictHostKeyChecking=no",
+        "torizon@${config:torizon_ip}",
+        "LOCAL_REGISTRY=${config:host_ip} TAG=arm docker-compose up -d __container__-debug"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "flame",
+        "color": "terminal.ansiYellow"
+      }
+    },
+    {
+      "label": "deploy-torizon-arm",
+      "detail": "hide",
+      "command": "scp",
+      "type": "process",
+      "args": [
+        "-i",
+        "${workspaceFolder}/.conf/id_rsa",
+        "-o",
+        "UserKnownHostsFile=/dev/null",
+        "-o",
+        "StrictHostKeyChecking=no",
+        "-P",
+        "${config:torizon_debug_ssh_port}",
+        "-pr",
+        "${workspaceFolder}/target/armv7-unknown-linux-gnueabihf/debug",
+        "torizon@${config:torizon_ip}:~/app"
+      ],
+      "dependsOn": [
+        "copy-docker-compose",
+        "pre-cleanup-arm",
+        "build-debug-arm",
+        "build-container-torizon-debug-arm",
+        "push-container-torizon-debug-arm",
+        "pull-container-torizon-debug-arm",
+        "run-container-torizon-debug-arm"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "flame",
+        "color": "terminal.ansiYellow"
+      }
+    },
+    {
+      "label": "wait-debugpy-start-arm",
+      "detail": "hide",
+      "command": "sleep",
+      "type": "process",
+      "args": ["2"],
+      "dependsOn": [
+        "validate-settings",
+        "validate-arch-arm",
+        "apply-torizon-packages",
+        "copy-docker-compose",
+        "pre-cleanup-arm",
+        "build-debug-arm",
+        "build-container-torizon-debug-arm",
+        "push-container-torizon-debug-arm",
+        "pull-container-torizon-debug-arm",
+        "run-container-torizon-debug-arm",
+        "wait-a-bit",
+        "pos-cleanup",
+        "deploy-torizon-arm",
+        "start-torizon-debug-arm"
```

Comment:
I believe this function is also missing from the templates. 

As the Torizon ARMv7 function on the `launch.json` file is not in the Go launch function, please also review all the functions on this `dependsOn` to make sure they are the ARMv7 equivalent to their respective ARMv8 function (the ones on the `dependsOn` from line 839 to line 852)